osgar/drivers/rosmsg.py

```diff
@@ -382,13 +382,13 @@ def parse_bucket(data):
 def parse_topic(topic_type, data):
     """parse general topic"""
     if topic_type == 'srcp2_msgs/Qual1ScoringMsg':
-        assert len(data) == 44, (len(data), data)
+        assert len(data) == 8, (len(data), data)
         size = struct.unpack_from('<I', data)[0]
         pos = 4
-        assert size == 40, size
-        # __slots__ = ['score','calls','total_of_types']
-        # _slot_types = ['int32','int32','int32[8]']
-        return list(struct.unpack_from('<II', data, pos))  # only score and calls
+        assert size == 4, size
+        # __slots__ = ['score']
+        # _slot_types = ['int32']
+        return struct.unpack_from('<I', data, pos)
```

Comment:
why, for the hell, they removed number of attempts?! Then you do not have confirmation of your report :(